scripts/plusplus.js

```diff
@@ -0,0 +1,73 @@
+/* 
+ * Custom ++/-- script to handle people ++'ing stuff.
+ * 
+ * Features
+ * 
+ *   Per user rate limiting
+ *   Tracking totals
+ *   
+ */
+
+module.exports = function (robot) {
+
+    /**
+     * ++ - Increment points of target
+     * 
+     * */
+    robot.respond('/(.*)\\+\\+/', function (msg) {
```

Comment:
Using `robot.respond`, owlbot will only respond to ++/-- when the message it `@owlbot something++` was this intentional? If not, using `robot.hear` will filter any message (ie `something++`)